### PR TITLE
Added Docstrings

### PR DESCRIPTION
**Pull request by Padmini-desai**
Closes: https://github.com/MasoniteFramework/core/issues/849

### CODE REVIEW

File: masonite/drivers/cache/CacheDiskDriver.py [hunk 1]
```diff
@@ -168,6 +168,14 @@ def is_valid(self, key):
...
 
     def _create_directory(self, directory):
+        """creates a new dir.
+
+               Arguments:
+                   directory {string} -- name of dir
+
+               Returns:
+                   bool|string -- Returns a boolean on new dir creation
+               """
```
Comment (vaibhavmule) [inline masonite/drivers/cache/CacheDiskDriver.py]: @Padmini-desai 
indent the docstring properly.

File: masonite/drivers/cache/CacheDiskDriver.py [hunk 2]
```diff
@@ -168,6 +168,14 @@ def is_valid(self, key):
         return False
 
     def _create_directory(self, directory):
+        """creates a new dir.
+
```
Comment (vaibhavmule) [inline masonite/drivers/cache/CacheDiskDriver.py]:
```suggestion

```

Comment (vaibhavmule) [reply masonite/drivers/cache/CacheDiskDriver.py]: @Padmini-desai  please remove the whitespace in the line number, 72

Comment (Padmini-desai) [reply masonite/drivers/cache/CacheDiskDriver.py]: @vaibhavmule removed the whitespaces 
Please review the same

Comment (vaibhavmule) [reply masonite/drivers/cache/CacheDiskDriver.py]: sure, looking at it.

**Comment by josephmancuso** [comment]
Great fix! Thank you!

**Pull request merged**